cmake/std/PullRequestLinuxDriver.sh

```diff
@@ -1,7 +1,44 @@
 #!/usr/bin/env bash
 # set -x  # echo commands
 
-#    make  sure we have a newer version of git/python
+function get_scriptname() {
+    # Get the full path to the current script
+    local script_name=`basename $0`
```

Comment:
Perhaps just realpath $0 ?

Reply:
I think I've run into compatibility issues on some platforms with `realpath` which is why I use `readlink -f` in other places. I can look into it, but if it's just cosmetic, I'd rather not make the change in this PR because the manual testing is a pain to run.